### PR TITLE
docs: quickstart update

### PR DESCRIPTION
**Pull request by hagen1778**
The change to QuickStart doc supposed to cover issues highlighted at https://github.com/VictoriaMetrics/VictoriaMetrics/issues/2267

There are two options how to see how the updated doc looks like:
1. Spin-up doc server locally (see Makefile cmds from in this PR)
2. Visit https://hagen1778.github.io/docs-test/Quick-Start.html and https://hagen1778.github.io/docs-test/keyConcepts.html

Signed-off-by: hagen1778 <roman@victoriametrics.com>

### CODE REVIEW

File: docs/Quick-Start.md [hunk 1]
```diff
@@ -4,74 +4,706 @@ sort: 13
...
+then its cardinality will be `3*5=15` of unique time series.
+
+Every time series consists of `datapoints` (also called `samples`). 
+A `datapoint` is value-timestamp pair associated with the specific series:
+```
+requests_total{path="/", code="200"} <float64 value> <unixtimestamp>
+```
+
+In VictoriaMetrics data model, datapoint's value is of type `float64`. 
+And timestamp is unixtime with milliseconds precision. Each series can contain infinite number of datapoints.
```
Comment (f41gh7) [inline docs/Quick-Start.md:134]: Should we mention series `step` there?

**Review by f41gh7 — APPROVED** [review]
LGTM

**Comment by valyala** [comment]
I'd create a separate doc such as `VictoriaMetrics fundamentals` with the added details and then refer to this dock from the `quick start` and other docs when necessary.
The `quick start` docs should be short and concise. Their main purpose is to guide the user on how to install VictoriaMetrics and start working with it. So let's split `quick start` doc into short `quick start` and long `victoriametrics fundamentals`

File: docs/Quick-Start.md (hunk 1 above)
Comment (hagen1778) [reply docs/Quick-Start.md:134]: With what reference do you think it should be mentioned?

**Comment by hagen1778** [comment]
@valyala see updated

**Review by dmitryk-dk — APPROVED** [review]
LGTM!

**Pull request merged**